src/components/account/AccountSelector.js

```diff
@@ -41,16 +41,16 @@ const AccountToggleItem = ({ account, balance }) => {
     );
   } else {
     ContentElement = (
-      <div style={{ marginLeft: caretSize + caretMargin, fontSize: '1.25rem' }}>
+      <div style={{ marginLeft: caretSize + caretMargin }}>
         Select Account
       </div>
     );
   }
   return (
     <>
-      <div className="account-toggle-item rounded border border-primary d-flex flex-row align-items-center p-2">
```

Comment:
Are these border styles kusama specific or do we want to remove them for polkadot as well?! I think we should plan to eventually have a single repo/code for both apps with some theme config and should be able to deploy each app simply by importing the correct .scss theme (static import per app is fine IMO). So if these border styles are needed for polkadot we want to add them to polkadot.scss.